src/components/VideoBlock/VideoBlock.tsx

```diff
@@ -115,12 +115,14 @@ const VideoBlock = (props: VideoBlockProps) => {
             return;
         }
 
-        const fullSrc = `${src}?${getPageSearchParams(attributes || {})}`;
-
         if (ref.current && !iframeRef.current) {
             const iframe = document.createElement('iframe');
             iframe.id = fullId;
-            iframe.src = fullSrc;
+
+            if (fullscreen && !previewImg) {
```

Comment:
why should we only use src when it is fullscreen, how about non-fullscreen cases?

Reply:
below in code we have check to show preview image and this is combined with fullSchreen. But ok, I guess you are right, we can remove this from condition and check only !previewImg